src/libs/OptionsListUtils.ts

```diff
@@ -1917,7 +1915,7 @@ function getOptions(
         }
     }
 
-    if (includePersonalDetails) {
+    if (includeP2P) {
```

Comment:
this condition will not be redundant? we're assigning an empty array above.

Reply:
@getusha You're right, I updated.

Reply:
@nkdengineer I am still not sure if we want to keep the code that was under this condition. could you please confirm that?

Reply:
@getusha The code that was under this condition is used to create personal detail option so it's necessary.

Reply:
@nkdengineer sorry, i was just confused because the code was supposed to run only under that condition right? removing only the condition didn't make any sense. and lets make sure we test this thoroughly.

Reply:
>     const havingLoginPersonalDetails = includeP2P ? options.personalDetails.filter((detail) => !!detail?.login && !!detail.accountID && !detail?.isOptimisticPersonalDetail) : [];

@getusha 

1. We update this to only filter `havingLoginPersonalDetails` if `includeP2P` is `true`

2. The code under the condition is calculated the personal detail option based on `havingLoginPersonalDetails` so we need to keep this code. we can remove the condition because we applied the condition at point 1.

Reply:
Makes sense, thanks